scripts/setup_ubuntu.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+DIR=$(cd "$(dirname "${BASH_SOURCE:-$0}")"; pwd)
+
+echo [$(date +"%F %T")] ================================
+echo [$(date +"%F %T")] Installing redis, node and python3.8
+echo [$(date +"%F %T")] ================================
+
+sudo apt-get update
+sudo apt-get install -y --no-install-recommends ca-certificates \
+    build-essential software-properties-common curl \
+    autoconf libtool pkg-config gnupg-agent git
+sudo add-apt-repository -y ppa:chris-lea/redis-server ppa:deadsnakes/ppa
+curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash -
+sudo apt-get update
+sudo apt-get install -y --no-install-recommends \
+    nodejs redis-server python3.8 \
```

Comment:
I get an error due to a bad version of Redis after running this script:
*** FATAL CONFIG FILE ERROR ***
Reading the configuration file, at line 308
>>> 'replica-serve-stale-data yes'
Bad directive or wrong number of arguments

It seems the script installed version 4.0.9 but we require 5.0.7. I fixed this when I was installing by using the wget command on the Redis website: https://redis.io/download

Reply:
With this method, I also had to modify the launcher to run the local Redis binary instead of the system-wide one.